Add option to skip current event after certain time (#40)
rosenpin/i3-agenda: README.md

                        max number of events to query Google's API for each of your calendars. Increase this number if you have lot of events in your google calendar
  --today, -d           print only today events
  --no-event-text TEXT  text to display when there are no events

>   --started-hide-minutes MINUTES, -s MINUTES

Reviewer: rename this argument `time-to-show-event` or `hide-event-after`\
the current name is unclear

Author: Thanks @rosenpin, I like `hide-event-after`, but the shortcut won't be mnemonic. Would it be OK if it doesn't have a shortcut?

Reviewer: that's fine

>                         minutes to wait before skiping the current event. If not specified, the current event will be showed until it ends

Reviewer: ```suggestion
                        minutes to show events after they start before showing the next event. If not specified, the current event will be shown until it ends

```
